wg-welcome/templates/project-specific-help.md

```diff
@@ -0,0 +1,31 @@
+# Question about a specific project
+
+## Question
+
+Sometimes people post to dev@community with a question about a specific
+Apache project. We are not the place for that question, and we need to
+gently redirect, while offering a little additional information about
+how the ASF works.
+
+## Answer
+
+Hi, _____, thanks for your interest in Apache.
+
+As you may know, The ASF is home to more than 200 projects, and each
+project has their own mailing lists, or other forums, for answering
+questions. This list - dev@community.apache.org - is about building
+communities across all these projects, but isn't usually the right place
+for project-specific questions.
+
+Your question is about the Apache [Foo] project, so the best place to seek
+advice is at https://[foo].apache.org/ or on their project-specific
+mailing lists at https://lists.apache.org/list.html?users@[foo].apache.org
+
+*** NOTE: Verify that the project in question has a users@ list, and
```

Comment:
2. Is this NOTE for the reply sender? Or it's included AS IS in the reply for the receiver.

Reply:
The note is for the sender. Ideally we would always edit these boilerplate messages, and customize them for each case.